### PR TITLE
1234567: Test merge between multiple local branches in personal fork

### PR DESCRIPTION
**Pull request by AlekseiEfimov**
---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [JDK-1234567](https://bugs.openjdk.java.net/browse/JDK-1234567): Date objects do not preserve milliseconds properly ⚠️ Title mismatch between PR and JBS.

### Reviewers
 * [Joe Wang](https://openjdk.java.net/census#joehw) (@JoeWang-Java - **Reviewer**)
 * aefimov - Committer ⚠️ Added manually

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/51/head:pull/51`
`$ git checkout pull/51`

### CODE REVIEW

File: mergeConflictFile.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is a change from 'BranchToMerge1'!
+Another line added by BranchToMerge2
+Completely ortogonal change from BranchToMerge3
+Just text
```
Comment (JoeWang-Java) [inline mergeConflictFile.txt:4]: The change looks great! :-)

**Review by JoeWang-Java — APPROVED** [review]

File: mergeConflictFile.txt (hunk 1 above)
Comment (AlekseiEfimov) [reply mergeConflictFile.txt:4]: Thanks for the review Joe :)

**Pull request closed without merging**